src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -0,0 +1,150 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { getCategory } from "../../utils/componentCategories";
+
+import DatePicker from "./DatePicker";
+import { DatePickerTypes } from "./DatePickerTypes";
+
+<Meta
+  title={getCategory("DatePicker")}
+  component={DatePicker}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=11689%3A0",
+    },
+    jest: ["DatePicker.test.tsx"],
+  }}
+  argTypes={{
+    blockName: { table: { disable: true } },
+    className: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+    id: { table: { disable: true } },
+    name: { table: { disable: true } },
+    value: { table: { disable: true } },
+  }}
+/>
+
+# DatePicker
+
+<Description of={DatePicker} />
+
+<Canvas withToolbar>
+  <Story
+    name="DatePicker"
+    args={{
+      dateType: DatePickerTypes.Full,
+      dateFormat: "yyyy-MM-dd",
+      dateRange: false,
+      name: "visit-dates",
+      labelText: "Select the date you want to visit NYPL",
+      helperText: "Note that the Library may be closed on Sundays.",
```

Comment:
does it make sense to merge generic `HelperText` and `HelperTextFrom`?  

(Eg: I imagine `HelperTextFrom` would have a smaller character limit than a generic `HelperText`, no?)

Reply:
Do you mean `HelperTextTo` instead of `HelperTextFrom`? It's possible but having two separate fields will be good for clearer instructions for what dates are expected, such as the figma file where it shows an early "From" date example and a later "To" date example:

<img width="371" alt="Screen Shot 2021-08-06 at 2 28 15 PM" src="https://user-images.githubusercontent.com/1280564/128555924-9419a1f1-3824-4960-b1a8-a406411803f7.png">

Reply:
I had a discussion with UX about using just one helper text value, but an explicit decision was made to use two separate helper text values.  Edwin nailed the reasoning in his explanation.

Reply:
Oh I'm wondering if there should be three `helperText` fields like, a `HelperTextFrom` `HelperTextTo` and generic `HelperText` 

Eg: for the example that Edwin posted, perhaps there's a need for a bigger `HelperText` underneath that says "Please note that books in the last ten years have not yet been digitized" or something 


It doesn't make sense to me that you can have a generic `HelperText` and a `HelperTextTo` together, but not `HelperText` and `HelperTextFrom` together.

Reply:
For the `dateRange` option, the `helperText` prop essentially acts like what you described as the "`helperTextFrom`" prop. It's implied so it can be confusing. For the moment, and based on the component spec, we're only going to use the `helperText` and `helperTextTo` props. I do agree that for some context/use-cases we may want another helper text.

We're continuously going to add addendums to the spec document and TAD document. If we all agree on this update, then we can add more information for this `helperTextFrom` prop update.

This is a new process so I'll set up a meeting to talk about this but to be brief:
- this PR covers the spec/functional requirements
- any updates or concerns should be added in the spec and TAD document in an "Addendum" or "Update" section
- the update would be a new ticket and PR